oak_runtime/src/node/wasm/tests.rs

```diff
@@ -73,15 +73,9 @@ fn start_node(
     result
 }
 
-fn create_signature(public_key_path: &str, signature_path: &str) -> Signature {
-    let public_key =
-        minisign::PublicKey::from_file(public_key_path).expect("Could not read public key file");
-    let signature =
-        minisign::SignatureBox::from_file(signature_path).expect("Could not read signature file");
-    Signature {
-        public_key,
-        signature,
-    }
+fn create_signature(signature_path: &str) -> Signature {
```

Comment:
I may be missing something, but it seems that this is loading a signature, not creating one?